test/case/ietf_syslog/remote/test.py

```diff
@@ -146,9 +145,11 @@
             }
         })
 
-    with test.step("Verify logging from client to server"):
-        clientssh.runsh("logger -t test -m client -p security.notice Hej")
+    with test.step("Send logging message with facility 'security' and severity 'notice' on client"):
```

Comment:
Maybe simplify to "Send security:notice log message from client", in particular *from* is more on point.